### PR TITLE
fix: use proper variable for title

### PR DESCRIPTION
**Pull request by kilatib**
# [TR-4814](https://oat-sa.atlassian.net/browse/TR-4814)

## Description 
The title property of the item is shown in the item list and as the tooltip text.

## Development impact
just use right property as on other places

## Demo 
https://user-images.githubusercontent.com/2750628/200305975-8c963282-d6c7-494c-b0b6-8d560b3f40f4.mov

### CODE REVIEW

**Review by poyuki — APPROVED** [review]
nice)

**Review by wazelin — APPROVED** [review]

**Review by wazelin — APPROVED** [review]

**Review by wazelin — APPROVED** [review]
There we go again.

**Review by wazelin — CHANGES_REQUESTED** [review]
Oh, now that you touched a new php file, could you also fix its code style?

The CI pipeline requires it nowadays.

```
----------------------------------------------------------------------
FOUND 13 ERRORS AND 1 WARNING AFFECTING 12 LINES
----------------------------------------------------------------------
   1 | ERROR   | [x] Header blocks must be separated by a single
     |         |     blank line
  50 | ERROR   | [ ] The first trait import statement must be
     |         |     declared on the first non-comment line after the
     |         |     class opening brace
 131 | ERROR   | [x] Blank line found at start of control structure
 139 | ERROR   | [x] Opening parenthesis of a multi-line function
     |         |     call must be the last content on the line
 142 | ERROR   | [x] Multi-line function call not indented correctly;
     |         |     expected 16 spaces but found 20
 142 | ERROR   | [x] Only one argument is allowed per line in a
     |         |     multi-line function call
 142 | ERROR   | [x] Closing parenthesis of a multi-line function
     |         |     call must be on a line by itself
 207 | ERROR   | [x] Multi-line function call not indented correctly;
     |         |     expected 28 spaces but found 32
 208 | ERROR   | [x] Multi-line function call not indented correctly;
     |         |     expected 28 spaces but found 32
 209 | ERROR   | [x] Multi-line function call not indented correctly;
     |         |     expected 24 spaces but found 28
 255 | ERROR   | [x] There must not be a space before the colon in a
     |         |     return type declaration
 261 | WARNING | [ ] Line exceeds 120 characters; contains 139
     |         |     characters
 309 | ERROR   | [x] Expected 1 space(s) after closing parenthesis;
     |         |     found 2
 313 | ERROR   | [x] Expected 1 space after closing brace; newline
     |         |     found
```

File: models/QtiRunnerInitDataBuilder.php [hunk 1]
```diff
@@ -197,15 +218,16 @@ protected function getTestMap(QtiRunnerServiceContext $context, array $itemsStat
                     $items[$itemId] = [
                         'id' => $itemId,
                         'label' => $itemData['data']['attributes']['label'],
+                        'title' => $itemData['data']['attributes']['title'],
```
Comment (kilatib) [inline models/QtiRunnerInitDataBuilder.php:221]: real change

**Review by poyuki — APPROVED** [review]

**Review by wazelin — APPROVED** [review]

**Pull request merged**